ivadomed/main.py

```diff
@@ -113,6 +113,9 @@ def film_normalize_data(context, model_params, ds_train, ds_valid, path_output):
     return model_params, ds_train, ds_valid, train_onehotencoder
 
 
+
+
+
```

Comment:
Do we need to add these 3 empty lines here?

Reply:
These are secret empty lines. I will remove it.